internal/dashboard/business/oceanbase/obtenant.go

```diff
@@ -244,7 +244,7 @@ func CreateOBTenant(ctx context.Context, nn types.NamespacedName, p *param.Creat
 			Namespace: nn.Namespace,
 		},
 		StringData: map[string]string{
-			"password": rand.String(20),
+			"password": p.RootPassword, // For simplicity, use the same password as root
```

Comment:
it's not a good practice to use the same password

Reply:
This is necessary if we require users to submit one password field.

Reply:
Otherwise, it brings problems when restoring a tenant from backup data. Currently it would generate a different password from original.

Reply:
If the user need to restore or create standby tenant, we can give them the way to retrieve this password, or we do it automatically, we can discuss later.